artifacts/requirements.md

```diff
@@ -6,8 +6,8 @@ Just copied the template for the requirements
 
 | Requirement ID | User Story ID | Requirement                                    | Acceptance Criteria                                                       | Effort | Priority  | Status   |
 | -------------- | ------------- | ---------------------------------------------- | ------------------------------------------------------------------------- | ------ | --------- | -------- |
-| 000            | 000           | The website must accept a username             | The site accepts the username                                             | 1      | Necessary | Verified |
-| 001            | 000           | The website must accept a password             | The site accepts the password                                             | 1      | Necessary | Verified |
-| 002            | 000           | The credentials correspond to a unique account | The login credentials are tied to a specific account                      | 2      | Necessary | Verified |
-| 003            | 003           | The website must have an easy to use interface | New users are able to navigate the website without issue                  | 3      | Necessary | Verified |
-| 005            | 005           | Live updates of the current stocks             | The site must have a constant update of data coming from the stock market | 5      | Necessary | Verified |
+| 000            | 004           | The website must accept a username             | The site accepts the username                                             | 1      | Necessary | Verified |
+| 001            | 004           | The website must accept a password             | The site accepts the password                                             | 1      | Necessary | Verified |
+| 002            | 004           | The credentials correspond to a unique account | The login credentials are tied to a specific account                      | 2      | Necessary | Verified |
+| 003            | 001           | The website must have an easy to use interface | New users are able to navigate the website without issue                  | 3      | Necessary | Verified |
+| 004            | 000           | Live updates of the current stocks             | The site must have a constant update of data coming from the stock market | 4      | Necessary | Verified |
```

Comment:
I think this last requirement is for user story [002]

Reply:
ok ill fix that and re push it as user story 002